mmf/trainers/base_trainer.py

```diff
@@ -494,8 +494,8 @@ def inference(self):
             self._inference_run("test")
```

Comment:
here can you add `"test" in self.run_type`?

Reply:
Actually, thinking more. I think run_type should be made a top level argument in defaults.yaml (at the same level as config). Why? Because it is being used by both evaluation and training and is not just about training.

Reply:
Ok I will move this to top level.